packages/compass-e2e-tests/helpers/commands/focus-stage-operator.js

```diff
@@ -0,0 +1,12 @@
+const Selectors = require('../selectors');
+
+module.exports = function (app) {
+  return async function focusStageOperator(index) {
+    const { client } = app;
+
+    await client.clickVisible(Selectors.stageCollapseButton(index));
+    await client.clickVisible(Selectors.stageExpandButton(index));
+    await client.keys(['Tab']);
```

Comment:
This is nuts but it really is as if react-select-plus was written in such a way that it is deliberately as hard to write e2e tests for as possible. I could only focus it by tabbing to it, not by clicking it or anything inside it. Yet obviously you can click it manually... After hours and hours this is the best I could come up with.